TESTOWNERS

```diff
@@ -86,7 +86,7 @@
 /dev/devicelab/bin/tasks/raster_cache_use_memory_perf__e2e_summary.dart @flar @flutter/engine
 /dev/devicelab/bin/tasks/routing_test.dart @zanderso @flutter/tool
 /dev/devicelab/bin/tasks/shader_mask_cache_perf__e2e_summary.dart @flar @flutter/engine
-/dev/devicelab/bin/tasks/spell_check_test.dart @camsim99 @flutter/android
+/dev/devicelab/bin/tasks/spell_check_test_ios.dart @camsim99 @flutter/android
```

Comment:
To avoid breakage, shall we keep these two entries? After the cocoon PR, we can remove `spell_check_test`.

Reply:
Ah yeah.  I can do this.